dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -205,6 +215,16 @@ if you only build few packages, run:
 breeze release-management prepare-provider-packages --package-format both PACKAGE PACKAGE ....
 ```
 
+NOTE! When you want to release a suspended provider, package for the provider will not be prepared
+when you prepare all packages - you have to specifically use the provider name in a separate
+command. For example to prepare documentation for suspended `qubole` provider you need to run
+separately this command:
+
+```shell script
+breeze release-management prepare-provider-packages --package-format both qubole
```

Comment:
What would be an event when you'd prepare the documents for the suspended providers? Security fixes?

Reply:
According to agreed process of provider's removal process - we should make one LAST release of the provider whch includes information that the provider is not maintained any more: https://github.com/apache/airflow/blob/main/PROVIDERS.rst#removing-community-providers

> One last release of the provider is done with documentation updated informing that the provider is no longer maintained by the Apache Airflow community - linking to this page. This information should also find its way to the package documentation and consequently - to the description of the package in PyPI.

Both in documentation and in README that goes to PyPI. This is why we want to be able to build and release package and documentation for the suspended provider. 

I also considered un-suspending provider, releasing and THEN removing, but that would be rather cumbersome and involves for example rebuilding the CI image with all the provider dependencies. I decided that allowing to build docs, publish them and release suspended provider is way more reasonable approach.

Reply:
But. I understand the question and see why it might be bad. Just to be on a safe side, I will add a "removed" flag that will enable this and add more description about it. Just to avoid confusion

Reply:
Thanks for the background here :) 
Also, I agree that adding an explicit `remove` would be much clearer.

Reply:
OK. I added a separate "remove" options (which means `scheduled for removal`) and now you should only  be able to release suspended  package when it is scheduled for removal. Also updated the docs generation so that in this case the "removal" warning is added automatically. And updated docs reflecting it in release proceedure. This way it is much more robust and `clean` process.